fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/Action.java

```diff
@@ -23,7 +23,11 @@ public interface Action {
 
     record TextAction(String text) implements Action {
         @Override
-        public void simpleVerify() {}
+        public void simpleVerify() {
+            if (this.text().contains("  ")) {
+                throw new FabledTechnicalException("Text contains double spaces. Replace all double spaces with a single space.");
```

Comment:
Pleas  replace throw new Error with an replace function

Reply:
sounds good, but in a follow up PR, it doesn't have to block this one.

Reply:
Just fix it now